File: main-3.py
def convert(n):
    return[int(num) for num in str(n)[::-1]]
 
print(convert(564987654))  # [4, 5, 6, 7, 8, 9, 4, 6, 5]
print(convert(529132))  # [2, 3, 1, 9, 2, 5]

File: main3.js
function convert(n) {
  let st = String(n);
  let emptyArray = [];
  for (let index = 0; index < st.length; index++) {
    emptyArray.push(+st[index]);
  }

  // return emptyArray.reverse();

  let result = emptyArray.reverse();
  return result;
}

console.log(convert(564987654)); // [4, 5, 6, 7, 8, 9, 4, 6, 5]
console.log(convert(529132)); // [2, 3, 1, 9, 2, 5]
